bmorph/core/workflows.py

```diff
@@ -480,11 +529,11 @@ def unpack_and_write_netcdf(results, segs, file_path, out_varname='scbc_flow', m
             config_dir=bmorph_config['data_path']+'/mizuroute_configs/',
             topo_dir=bmorph_config['data_path']+'/topologies/',
             input_dir=bmorph_config['data_path']+'/input/',
-            output_dir=bmorph_config['data_path']+'/output/',
+            output_dir=bmorph_config['data_path']+bmorph_config['output_sub'],
             )
 
     mizutil.run_mizuroute(mizuroute_exe, config_path)
-    region_totals = xr.open_mfdataset(f'{mizuroute_config["output_dir"]}{bmorph_config["output_prefix"]}_{scbc_type}_scbc*')
+    region_totals = xr.open_mfdataset(f'{mizuroute_config["output_dir"]}{bmorph_config["output_prefix"]}_{scbc_type}_scbc*', engine='netcdf4')
```

Comment:
Was this needed?

Reply:
I was getting errors that it didn't know the file type and putting this in fixed them. Uncertain if they were purely a result of the file corruption issue or not.

Reply:
I can try removing it and running things again shortly

Reply:
not needed, will remove